pkgs/development/libraries/x265/default.nix

```diff
@@ -34,9 +34,13 @@ let
     sha256 = "1jzgv2hxhcwmsdf6sbgyzm88a46dp09ll1fqj92g9vckvh9a7dsn";
   };
 
+  patches = stdenv.lib.optionals stdenv.hostPlatform.isAarch64 [
```

Comment:
patches shouldn't be conditional on a specific architecture - assuming the patch won't apply after a bump, this will lead to silent breakages.

If this patch doesn't break other arches, it should just be applied independent of architecture